R/dataProcessPlots.R

```diff
@@ -129,20 +136,92 @@ dataProcessPlots = function(
+                  og_plotly_plot = .fix.legend.plotly.plots.dataprocess(og_plotly_plot)
+                  og_plotly_plot = .fix.censored.points.legend.profileplots.plotly(og_plotly_plot)
+
+                  if(toupper(featureName) == "NA") {
+                      og_plotly_plot <- og_plotly_plot %>% style(og_plotly_plot, showlegend = FALSE)
+                  }
+                  plotly_plots = c(plotly_plots, list(og_plotly_plot))
+              }
+          }
+          if(rlang::has_name(plots, "summary_plot")) {
+              for(i in seq_along(plots[["summary_plot"]])) {
+                  plot_i <- plots[["summary_plot"]][[paste("plot",i)]]
+                  summ_plotly_plot <- .convert.ggplot.plotly(plot_i,tips=c("FEATURE","RUN","ABUNDANCE"))
+                  summ_plotly_plot = .fix.legend.plotly.plots.dataprocess(summ_plotly_plot)
+                  summ_plotly_plot = .fix.censored.points.legend.profileplots.plotly(summ_plotly_plot)
+                  if(toupper(featureName) == "NA") {
+                      summ_plotly_plot <- summ_plotly_plot %>% style(summ_plotly_plot, showlegend = FALSE)
+                  }
+                  plotly_plots = c(plotly_plots, list(summ_plotly_plot))
+              }
+          }
+          
+          if(address != FALSE) {
+              .save.plotly.plot.html(plotly_plots,address,"ProfilePlot" ,width, height)
+          }
+          return(plotly_plots)
+      }
+  }
+    
+  if (type == "QCPLOT") {
+      plots <- .plotQC(processed, featureName, ylimUp, ylimDown, x.axis.size, y.axis.size, 
+                      text.size, text.angle, legend.size, dot.size.profile, width, height,
+                      which.Protein, address, isPlotly)
+      plotly_plots <- list()
+      if(isPlotly) {
+          for(i in seq_along(plots)) {
+              plot <- plots[[i]]
+              plotly_plot <- .convert.ggplot.plotly(plot)
+              plotly_plot = .fix.legend.plotly.plots.dataprocess(plotly_plot)
+              plotly_plots = c(plotly_plots, list(plotly_plot))
+          }
+            
+            if(address != FALSE) {
+                .save.plotly.plot.html(plotly_plots,address,"QCPlot" ,width, height)
+            }
+            return(plotly_plots)
+      }
+  }
+    
+  if (type == "CONDITIONPLOT") {
+      plots <- .plotCondition(processed, summarized, ylimUp, ylimDown, scale, interval,
+                             x.axis.size, y.axis.size, text.size, text.angle, legend.size, 
+                             dot.size.profile, dot.size.condition, width, height,
+                             which.Protein, save_condition_plot_result, address, isPlotly)
+      plotly_plots <- list()
+      if(isPlotly) {
+          for(i in seq_along(plots)) {
+              plot <- plots[[i]]
+              plotly_plot <- .convert.ggplot.plotly(plot)
+              plotly_plot = .fix.legend.plotly.plots.dataprocess(plotly_plot)
```

Comment:
is this naming convention necessary?

Reply:
Commit [better function names](https://github.com/Vitek-Lab/MSstats/pull/107/commits/8522cb8bd56359ae86602aecc388f9a320bd688b)